test/StringConversion.test.cpp

```diff
@@ -11,11 +11,9 @@ TEST(StringConversion, ToLower)
 TEST(StringConversion, ToLowerInPlace)
 {
 	std::string str("TEST");
-	ToLowerInPlace(str);
-	EXPECT_EQ("test", str);
+	EXPECT_EQ("test", ToLowerInPlace(str));
```

Comment:
You may want to check both the return value and the original value is modified:
```cpp
EXPECT_EQ("test", ToLowerInPlace(str));
EXPECT_EQ("test", str);
```

Without both checks, the `ToLower` method could be used to satisfy the test, even though it offers different behavior from `ToLowerInPlace`.